.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
@bewuer please remove IDEA files from pull request

Reply:
added .idea and build to gitignore